src/scripts/common.js

```diff
@@ -509,7 +509,10 @@ window.togglbutton = {
     }
 
     link.innerHTML = togglButtonSVG;
-    setLinkText(link, 'Start Timer');
+
+    if (params.buttonType !== 'minimal') {
```

Comment:
There's an if for the same check a few lines above, maybe we can use it instead of adding a new one 🤔

Reply:
Oh no nevermind it's `!==` 🤦‍♂️